README.md

```diff
@@ -1232,12 +1232,12 @@ _See also: [List of streaming media systems - Wikipedia](https://en.wikipedia.or
 - [Bsimp](https://github.com/akrylysov/bsimp) - Minimalistic S3-backed audio library. `Apache-2.0` `Go`
 - [euterpe](https://listen-to-euterpe.eu) - Self-hosted music streaming server with RESTful API and Web interface. ([Demo](https://listen-to-euterpe.eu/demo), [Source Code](https://github.com/ironsmile/euterpe)) `GPL-3.0` `Go`
 - [Funkwhale](https://dev.funkwhale.audio/funkwhale) - Modern, web-based, convivial, multi-user and free music server. `BSD-3-Clause` `Python/Django`
-- [GNU FM](https://gnu.io/fm/) - Running music community websites, alternative to last.fm. ([Source Code](https://git.savannah.gnu.org/cgit/librefm.git/)) `AGPL-3.0` `PHP`
+- [GNU FM](https://gnu.io/fm/) - Running music community websites (alternative to last.fm). ([Source Code](https://git.savannah.gnu.org/cgit/librefm.git/)) `AGPL-3.0` `PHP`
```

Comment:
> https://fedi.network/fm

Seems to be a defunct project. The site points users to https://github.com/foocorp/hacienda

Reply:
Yes I also noticed this. I opened https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/39 that removes it, but we will have to wait for the full switch to https://github.com/awesome-selfhosted/awesome-selfhosted-data first.